src/clang-fetcher.ts

```diff
@@ -50,6 +51,45 @@ export async function getClangEnvironmentVars(electronVersion: string, targetArc
   if (process.platform === 'linux') {
     const sysrootPath = await downloadLinuxSysroot(electronVersion, targetArch);
     clangArgs.push('--sysroot', sysrootPath);
+
+    const cc = `"${path.resolve(clangDir, 'clang')}" ${clangArgs.join(' ')}`
+    const cxx = `"${path.resolve(clangDir, 'clang++')}" ${clangArgs.join(' ')}`
+
+    // on Electron 13 or higher, build native modules with Electron's libc++ libraries
+    if (parseInt(electronVersion.split('.')[0]) >= 13) {
+      const libcxxObjects = await downloadLibcxxObjects(electronVersion, targetArch)
+      const libcxxHeadersDownloadDir = await downloadLibcxxHeaders(electronVersion, 'libc++')
+      const libcxxabiHeadersDownloadDir = await downloadLibcxxHeaders(electronVersion, 'libc++abi')
+
+      const libcxxHeaders = path.resolve(libcxxHeadersDownloadDir, 'include')
+      const libcxxabiHeaders = path.resolve(libcxxabiHeadersDownloadDir, 'include')
+
+      const cxxflags = [
+        '-nostdinc++',
+        '-D_LIBCPP_HAS_NO_VENDOR_AVAILABILITY_ANNOTATIONS',
```

Comment:
FYI: with roll https://github.com/electron/electron/pull/29895 , this macro should be removed in favor of including the `__config_site` file from the root of `libcxx` header file. https://github.com/electron/electron/pull/29895/commits/797723ec838709ddeba0c104e30727ee0b7ac8ca